README.md

```diff
@@ -25,78 +18,164 @@ None
 
 | Variable | Description | Default |
 |----------|-------------|---------|
-| unbound\_user | user of unbound | {{ \_\_unbound\_user }} |
-| unbound\_group | group of unbound | {{ \_\_unbound\_group }} |
-| unbound\_service | service name | unbound |
-| unbound\_conf\_dir | path to dir of config directory | {{ \_\_unbound\_conf\_dir }} |
-| unbound\_conf\_file | path to `unbound.conf(5)` | {{ \_\_unbound\_conf\_dir }}/unbound.conf |
-| unbound\_flags | unused | "" |
-| unbound\_script\_dir | directory to install scripts in `files` | {{ \_\_unbound\_script\_dir }} |
-| unbound\_directory | work directory | {{ \_\_unbound\_directory }} |
-| unbound\_config\_chroot | path to chroot directory | "" |
-| unbound\_freebsd\_chroot\_devfs\_ruleset\_number | `devfs(8)` rule set number. Change when unbound\_config\_chroot is not empty and you have other `devfs(8)` rule set with the same number. | 100 |
-| unbound\_script\_dir | directory to keep support script. this must be included in PATH environment variable. | {{ \_\_unbound\_script\_dir }} |
-| unbound\_config\_interface | `interface` to listen on | [] |
-| unbound\_config\_outgoing\_interface | `outgoing-interface` | "" |
-| unbound\_config\_do\_not\_query\_localhost | `do-not-query-localhost` | yes |
-| unbound\_config\_do\_ip4 | `do-ip4` | yes |
-| unbound\_config\_do\_ip6 | `do-ip6` | no |
-| unbound\_config\_access\_control | `access-control` | [] |
-| unbound\_config\_hide\_identity | `hide-identity` | yes |
-| unbound\_config\_hide\_version | `hide-version` | yes |
-| unbound\_config\_use\_syslog | `use-syslog` | yes |
-| unbound\_config\_private\_address | `private-address` | ["10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16", "192.254.0.0/16", "fd00::/8", "fe80::/10"] |
-| unbound\_config\_private\_domain | `private-domain` | [] |
-| unbound\_config\_remote\_control\_control\_enable | `control-enable` | yes |
-| unbound\_config\_remote\_control\_control\_use\_cert | `control-use-cert` | no |
-| unbound\_config\_remote\_control\_control\_interface | `control-interface` | "" |
-| unbound\_config\_server\_key\_file | `server-key-file` | {{ unbound\_config\_directory }}/unbound\_server.key |
-| unbound\_config\_server\_cert\_file | `server-cert-file` | {{ unbound\_config\_directory }}/unbound\_server.pem |
-| unbound\_config\_control\_key\_file | `control-key-file` | {{ unbound\_config\_directory }}/unbound\_control.key |
-| unbound\_config\_control\_cert\_file | `control-cert-file` | {{ unbound\_config\_directory }}/unbound\_control.pem |
-| unbound\_forward\_zone | `forward-zone` | [] |
+| `unbound_user` | user of `unbound` | `{{ __unbound_user }}` |
+| `unbound_group` | group of `unboun` | `{{ __unbound_group }}` |
+| `unbound_service` | service name of `unbound` | `unbound` |
+| `unbound_conf_dir` | path to config directory | `{{ __unbound_conf_dir }}` |
+| `unbound_conf_file` | path to `unbound.conf(5)` | `{{ __unbound_conf_dir }}/unbound.conf` |
```

Comment:
s/__unbound_conf_dir/unbound_conf_dir/ or users must modify unbound_conf_file when unbound_conf_dir is set.